domainobjs/ts/ballot.ts

```diff
@@ -0,0 +1,128 @@
+import assert = require("assert");
```

Comment:
```suggestion
import assert from "assert";
```

Reply:
These imports still have strange mixture of require and import.

Reply:
> These imports still have strange mixture of require and import.

Our convention should be to always use import, ya?